ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/fonts/PaywallFont.kt

```diff
@@ -0,0 +1,57 @@
+package com.revenuecat.purchases.ui.revenuecatui.fonts
+
+import android.os.Parcelable
+import androidx.annotation.FontRes
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import kotlinx.parcelize.Parcelize
+import kotlinx.parcelize.TypeParceler
+
+/**
+ * Represents a font. You can create either a [GoogleFont] or a [ResourceFont].
+ */
+sealed class PaywallFont : Parcelable {
+    /**
+     * Represents a downloadable Google Font.
+     */
+    @Parcelize
+    data class GoogleFont(
```

Comment:
I actually wondered whether we wanted to offer a similar API for compose... But I think it's ok to leave the responsibility of building the FontFamily (from any source) to the developer

Reply:
What would that look like?

Reply:
Basically it would be to allow using a `ParcelableFontProvider` in `PaywallViewOptions`, so both using a GoogleFont or a `ResourceFont` are allowed... But again, I think if they are using a custom font from those places, it's easier for them to just pass us the `FontFamily` directly in compose world, where there is no need for this data to be parcelable.